### DISCUSSION TITLE
Clone, Commit, Conquer: Your New to GitHub Community Check-In 💪

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#146692] [New to GitHub]
Hey hey! 👋 Reporting in for a New to GitHub Community Check-In 🗞️

## 🏆​​ Top Contributors in the Community
Big Kudos to these Superstar Community Contributors 👏
- Shoutout to @PatchaIT for working out loud and finding an answer to their question: [Workflow on release publish doesn't trigger for new releases targeting other branches than main](https://github.com/orgs/community/discussions/144130). Although a community member did step in to help, this user was able to hone in and find the answer on their own 🎉
- Thank you @AuroraEchos for providing guidance on why @vsbel may be running into [issues with adding wikis to their private repository in their organization](https://github.com/orgs/community/discussions/144984). 
- Looking for an IDE recommendation? @CyberSphinxxx shared their opinion on [which IDE they felt was the best](https://github.com/orgs/community/discussions/145986). Have a differing opinion? Feel free to make your case in the discussion as well! 

## 🧠 Popular Unanswered Posts in the Community
Got your thinking cap on? These popular community posts could use your expertise!

- We’ve got a user wondering [if it is possible to make a campus navigation where the map is offline, and is it possible to make the app using html and css](https://github.com/orgs/community/discussions/145449). Got any ideas on how to do so? Please chime in! 
- There’s an issue where a user [added a large file to local and now I can not push](https://github.com/orgs/community/discussions/145940)

Have you been helped by a community superstar that you’d like to see celebrated? Tag them to nominate them below!

For community members who post questions, [marking responses that solves your question as the answer](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer) goes a long way. When you mark a comment in the Discussion as an answer, GitHub will highlight the comment to help other community members quickly find the answer as well.
![The image is a screenshot showing where users can click to mark a response as the answer](https://github.com/user-attachments/assets/d874411b-a558-4151-9285-e45d5c1195a5)

Remember to thank these and other community members that take the time to comment and help troubleshoot these problems. An acknowledgement of a job well done can have an incredible effect and make someone’s day. ☀️ We’re here to learn, discover and connect so try and mark a comment as an answer if suitable, give a thank you and select a reaction as a way to show your appreciation! 

## 🪐✨Universe 2024 - Did You Miss It? 
The dust has settled, but the excitement is far from over! Universe 2024 brought together developers, creators, and innovators to explore the future of software development—and we’re here to make sure you didn’t miss a thing.

From groundbreaking product announcements to deep dives into security, AI, and collaboration tools, Universe 2024 was packed with insights and inspiration. Whether it was a sneak peek into GitHub’s next-gen features or hands-on sessions with industry leaders, there was something for everyone.

- [Complex migrations: Why failing forward helps push past resistance to change](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715190440825001r1em)
- [Contributing with confidence: Capital One's open source contribution workflows](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715458536520001tfIR)
- [Don't observe, debug: Discover how to quickly trace issues](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1719256171543001hV68)
- [From migration to mastery: Building an engineering stack with GitHub in six months](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1717424513668001OVJ5)
- [The sky’s the limit: Supercharge innovation through GitHub Certifications](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715357486644001k0ph)

Explore the highlights via the On Demand catalog here: https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog?search.deliveryformat=1692799009854004CQlx. 

## 🌳 Putting Down Roots: Nurturing Growth in Tech
Ideas that will help lay strong foundations, nurture ideas, and foster an environment conducive to personal and professional development and technological advancement.
- **[Finish your projects](https://github.com/readme/guides/finish-your-projects)** by Aaron Francis - Don’t let fear, or that last 10%, hold you back. 
- **[Build your own luck](https://github.com/readme/podcast/functional-programming)** - Our ReadME Project hosts discuss the value of taking risks and publishing your work 💪
- **[You can really do anything you want.. with transferable skills!](https://github.com/orgs/community/discussions/109396)** by Jenny Ngo - Learn more about how @hijennyngo successfully transitioned from a non-technical background to a technical career.
- **[Junior to senior: An action plan for engineering career success](https://github.com/readme/guides/engineering-career-success)** by Jerome Hardaway - Enhance your engineering career prospects using this systematic guide for skill development and growth.
- **[The importance of non-code contributions](https://github.com/orgs/community/discussions/58571)** by Kevin Sundstrom - From documentation to release management, non-code contributions power open source. 
- **[Intentional creation](https://github.com/readme/guides/intentional-creation)** by Mahmoud Hashemi - Learn how to tap into creativity with the 4 Cs: Consume, critique, curate, create.
- **[Making operational work more visible](https://github.com/readme/guides/ops-work-visible)** by Lorin Hochstein - How to shoulder-surf remotely and learn from the experiences of others.

## ✍️ Quote of the Month
_Practically, intentional creation means consciously spending less time on consumer sites, from Twitter to Hacker News, and more time taking notes, tagging bookmarks, and creating your own knowledge base. Attempt activities that are less entertainment and more you, ultimately closing the gap between you and your creative goals. _
  - From [Intentional Creation](https://github.com/readme/guides/intentional-creation) by Mahmoud Hashemi

**Comment by ghostinhershell** [comment]
Hey there! 👋🏾

Missed out on our last community check-in? You can check that out here: [**Putting a Spotlight on this Community Check-In 🌟**](https://github.com/orgs/community/discussions/142422).

**Comment by PatchaIT** [comment]
Thank you for the mention, but that wasn't neither the hardest question that I had to dig in hard to end up finding the answer by myself! 😅

**Comment by Rochdi7** [comment]
Hey everyone! 👋 Big shoutout to **@PatchaIT, @AuroraEchos, and @CyberSphinxxx** for their awesome contributions! 🎉
